google-ads/src/test/java/com/google/ads/googleads/lib/utils/AbstractErrorUtilsTest.java

```diff
@@ -200,4 +225,118 @@ protected ErrorPath<MockError> createErrorPath(MockError error, MockPath errorLo
+              // Gets the type of the request parameter.
+              .map(m -> m.getParameterTypes()[0])
+              // Invokes the getDescriptor method for easier inspection of the request type.
+              .map(
+                  msgClass -> {
+                    try {
+                      return (Descriptor) msgClass.getMethod("getDescriptor").invoke(new Object[0]);
+                    } catch (ReflectiveOperationException refExc) {
+                      throw new RuntimeException(
+                          "Failed to invoke getDescriptor on: " + msgClass, refExc);
+                    }
+                  })
+              // Keeps the request descriptor only if it is not one of the known exceptions.
+              .filter(desc -> !requestDescriptorsToIgnore.contains(desc))
+              // Keeps the request descriptor if it contains at least one repeated field. Request
+              // types with no repeated fields cannot have a list of operations, so they are
+              // irrelevant for this test.
+              .filter(desc -> desc.getFields().stream().anyMatch(fldDesc -> fldDesc.isRepeated()))
+              // Keeps the request descriptor if it does not have any fields that match the known
+              // list of operation field names.
+              .filter(
+                  desc ->
+                      desc.getFields().stream()
+                          .noneMatch(fldDesc -> OPERATION_FIELD_NAMES.contains(fldDesc.getName())))
+              // Gets the name of the request type.
+              .map(desc -> desc.getName())
+              // Adds all unmapped request type names to the list for this service client.
+              .collect(Collectors.toList());
+
+      // Only adds an entry for the service client if there are unmapped request types.
+      if (!unmappedRequests.isEmpty()) {
+        unmappedRequestsByService.putAll(serviceClientType.getName(), unmappedRequests);
+      }
+    }
+
+    if (!unmappedRequestsByService.isEmpty()) {
+      // Fails the test with a detailed error message containing all request types that are not
+      // properly handled by the error utility.
+      StringBuilder failureBuilder = new StringBuilder();
+      failureBuilder
+          .append(
+              "No known operations field found for the following service/request combinations: ")
+          .append(String.format("%n"))
+          .append(
+              unmappedRequestsByService.entries().stream()
+                  .map(entry -> entry.getKey() + "." + entry.getValue())
+                  .collect(Collectors.joining(String.format("%n"))))
+          .append(String.format("%n"))
+          .append(
+              "Review each service/request combination. If the request does not contain an"
+                  + " operations list, add the request to this test's list of request descriptors"
+                  + " to ignore. Otherwise, update the ErrorPath implementation to properly handle"
```

Comment:
It might be helpful to point to OPERATION_FIELD_NAMES in the error util and parameters above so folks know what to update in the event this comes up.

Reply:
Good point. Updated the failure message here.